README.md

```diff
@@ -92,3 +85,16 @@ const server = new ApolloServer({ schema });
 
 server.listen().then(({ url }) => console.log(`Listening on port ${url}`));
 ```
+
+If you need to merge typeDefs other than `Query`, `Mutation`, and `Subscription` definitions something you can pass in your own custom type(s) by passing in a string or an array of strings
+
+```js
+const gql = require('graphql-tag')
+const { mergeTypeDefs } = require('graphql-tools-merge-typedefs');
+
+const typeDefs = mergeTypeDefs([
+  customerSchema.typeDefs,
+  ...orderSchema.typeDefs
```

Comment:
Again, not sure why we're expanding the order schema's typeDefs but not the customer schema's.